nginx/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN apt-get update && \
   apt-get clean && \
   rm -r /var/cache/apt /var/lib/apt/lists
 COPY --from=buildstep /usr/local/nginx /usr/local/nginx
-COPY --from=nginx/nginx-prometheus-exporter /usr/bin/exporter /nginx-prometheus-exporter
+COPY --from=nginx/nginx-prometheus-exporter /usr/bin/nginx-prometheus-exporter /nginx-prometheus-exporter
```

Comment:
Please append the version tag to the image reference. Like this:
`nginx/nginx-prometheus-exporter:0.9.0`